CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+
+## v1.0.5
+### 2024-08-19
+
+This version of HOSS updates the 'is_index_subset' method to check for empty list (in case of dimensions subset) 
+as well as None (for the spatial, bbox and temporal subsets. This prevents prefetch from being called for 
```

Comment:
Other nit: This line is missing a closing parenthesis. I think it's meant to be:

> as well as `None` (for spatial, bounding box and temporal subsets). This prevents prefetch from being called for

I guess one other note: I don't know that "prefetch" is too informative for a lot of folks. Maybe:

> This prevents 1-D dimension variables from being unnecessarily requested from OPeNDAP for some variable subsets, in particular SMAP L3 and L4 collections.

Nit on line below: "occuring" -> "occurring".